docs/features/spreadsheets/README.md

```diff
@@ -0,0 +1,88 @@
+---
+category: inserting-content
+order: 70
+url: features/spreadsheets
+menu-title: Spreadsheets
+meta-title-short: Spreadsheet plugin
+---
+<!--
+Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md.
+-->
+
+# Spreadsheet plugin features overview
+
+<info-box info="">
+	This feature is compatible with **CKEditor 4** starting from `4.13` version. It is provided through an optional plugin that is not included in the CKEditor presets available from the [Download](https://ckeditor.com/ckeditor-4/download/) site. Follow the {@link guide/dev/integration/spreadsheets/README Spreadsheets Integration} guide to enable it.
+</info-box>
+
+The optional [Spreadsheet](https://ckeditor.com/cke4/addon/spreadsheet) plugin lets you insert customizable spreadsheet widgets, that has the following capabilities:
+
+* Inserting spreadsheets with and without header row/column and with any number of rows/columns.
+* Inserting spreadsheets using predefined templates (see {@link guide/dev/integration/spreadsheets/README#templates-plugin-integration Content Template plugin integration guide}).
+* Converting existing tables to spreadsheet instances and vice versa.
+* Single and multi-column sorting.
+* Basic data styling like bold, italics, underline and so on.
+* Defining data types and formatting with support for validation invalid data.
+* Inserting formulas.
+* Applying various conditional formatting rules to single/multiple cells, entire columns or spreadsheets.
+
+## Inserting spreadsheet widget
+
+Spreadsheet widget can be inserted via spreadsheet dialog or using Content Templates plugin with prepared spreadsheet templates. Below you can see simple 3x3 spreadsheet widget right after being inserted into editor.
+
+{@img assets/img/spreadsheets1.png Spreadsheet plugin widget}
+
+## Converting existing tables
+
+If you already have tables in your content and plan to use Spreadsheet plugin there is no need to recreate entire table from scratch as you may simply convert them to spreadsheet widget instance:
+
+{@img assets/img/spreadsheets2.png Spreadsheet plugin convert existing table context menu option}
+
+## Sorting
+
+Sorting is a must to efficiently analyze and present any tabular data, thus spreadsheet plugin allows for sorting data based on single or multiple columns:
+
+{@img assets/img/spreadsheets3.png Spreadsheet plugin widget with multisort}
+
+## Basic styles support
+
+While spreadsheets are mostly about data, its presentation is also very important. Basic styles like bold or italics comes handy when trying to highlight or make more visible some parts of data or single spreadsheet cells:
```

Comment:
There is also justify and removeformat support but not sure if it's worth pointing this out / treat as overall basicstyles support :thinking: